example/02_Components/Displaying Content/List/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "authors": [
+    "barb@hooli.com"
+  ],
+  "description": "A List-component displays items in an ordered or unordered list.\n",
+  "tags": [
+    "Progress/Draft",
+    "Release/0.6"
+  ],
+  "version": 1,
+  "custom": {
+    "synonyms": [
+      "Listing",
+      "Index"
+    ]
+  }
+}
```

Comment:
I think this is a leftover from an inbetween stage, when there was not yet YAML support on the feature. Can you restore the YAML (below) and add the custom keys there, then remove the JSON representation?

Reply:
done, was not sure, but decided to keep the custom data example